lib/core/src/display/camera/camera2d.rs

```diff
@@ -144,12 +149,16 @@ impl Camera2dData {
         let transform_dirty        = TransformDirty2::new(logger.sub("transform_dirty"),());
         let transform_dirty_copy   = transform_dirty.clone();
         let transform              = DisplayObjectData::new(logger);
-        let zoom_uniform           = globals.add_or_panic("zoom",1.0);
+        let zoom_update_callback   = Box::new(|_| ());
```

Comment:
Consider usage of Option<Box<ZoomUpdateFn>> - that type actually gives a information that the callback is optional, so it's set only in some cases. That's the way @mwu-tow prefer.